WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -818,4 +818,12 @@ public static void scrollIntoView(Integer scrollableContainerID, ViewInteraction
             swipeCount += 1;
         }
     }
+
+    public static void dismissJetpackAdIfPresent() {
+        // Dismiss Jetpack ad that might be shown after sign-up
+        // or after opening Stats
```

Comment:
Minor (🔍): Consider fitting this double-line comment in one line. FYI: The character limit for a line is `120`.

Reply:
Thanks, @ParaskP7 ! I've changed the comment to fit one line in 311ecba74760c226b13a52e80c07315e07a9848d. Waiting for the CI to turn 🟢 🙏 .